.devcontainer/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# The base container already has entrypoint, vscode user account, etc. out of the box
```

Comment:
This PR is about updating dependencies to be able to run it locally or build a release container?

In both cases I would suggest to move the devcontainer setup to a dedicated PR, as it is not related to the fix.